docs/connector-development/cdk-python/schemas.md

```diff
@@ -25,3 +25,85 @@ def get_json_schema(self):
     return schema
 ```
 
+## Schema normalization
```

Comment:
Unfortunately the world `normalization` is [already taken](https://docs.airbyte.io/understanding-airbyte/basic-normalization) in Airbyte 😅 

maybe we can replace it with something like `TypeTransformation` or `TypeCasting`?

Reply:
Done, replaced it with "type transformation"